gatsby/content/projects/other/matrix-quicksetup.mdx

```diff
@@ -0,0 +1,12 @@
+---
+layout: project
+title: matrix-quicksetup
+categories:
+ - other
```

Comment:
@imbev Any particular reason for closing?

This seems kinda in line with other projects in the `other` category like `matrix-docker-ansible-deploy` and docker-riot-web.

Reply:
I watched the matrix live video of last week and realized that matrix-quicksetup needs more work before release.